content/millennium/dstu2/general-clinical/procedure.md

```diff
@@ -36,11 +56,25 @@ Notes:
 
   - The `date` parameter must have a time, may be provided up to two times, and must use the `ge` or `le` prefixes. When provided twice, the lower value must have the `ge` prefix and the higher value must have the `le` prefix.
 
-### Response
+### Headers
+
+ <%= headers %>
+
+### Example
+
+#### Request
+
+    GET https://fhir-open.sandboxcerner.com/dstu2/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Procedure?patient=1316020
+
+#### Response
 
 <%= headers status: 200 %>
 <%= json(:dstu2_procedure_bundle) %>
 
+### Errors
+
+The common [errors] may be returned.
```

Comment:
Does it make sense to link all the common errors when some aren't accessible for a given resource? (409/422 are only for update/create etc.)